fixtures/hsl/40w-beam-spot-light-rgbw.json

```diff
@@ -0,0 +1,123 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "40W Beam Spot Light RGBW",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["David Oleniacz"],
+    "createDate": "2023-04-19",
+    "lastModifyDate": "2023-04-19"
+  },
+  "links": {
+    "productPage": [
+      "https://www.amazon.com/stores/HighIightSpIendidLife/page/C6F4337A-41CC-4560-880E-06BCA9E96924"
```

Comment:
This links takes user to the company's amazon storefront and not the light itself.
Link to product: https://www.amazon.com/Stage-Spotlight-Beam-Light-Package/dp/B07XYYK28H